lms/templates/learner_dashboard/expired_notification.underscore

```diff
@@ -0,0 +1,2 @@
+<span><i class="fa fa-info-circle fa-lg" aria-hidden="true"></i></span>
+<span>You were enrolled in this course previously but did not earn the certificate required to complete this program.</span>
```

Comment:
This message should reflect the possibility that the user remains enrolled in the course. Telling them that they "were enrolled" could confuse them if their dashboard indicates they remain enrolled. For example:

> You enrolled in this course but did not earn the certificate required to complete the program.